Hardware/Hardware/CardReader/CardBrand.swift

```diff
@@ -60,7 +66,7 @@ extension CardBrand {
             return "diners"
         case .interac:
             return "interac"
-        case .unknown:
+        case .unknown, .unionPay, .eftposAu:
```

Comment:
I believe we don't support it, but we seem to have an icon for the `.unionPay` on `WCPayCardBrand.unionpay`:
```
case .unionpay:
  return iconName(suffix: "unionpay")
```